content/doc/administration/requirements/jenkins-on-java-11.adoc

```diff
@@ -0,0 +1,56 @@
+---
+layout: documentation
+title: Running Jenkins on Java 11
+---
+
+If you need to upgrade the JVM version used to run Jenkins from Java 8 to Java 11, please link:/doc/administration/requirements/upgrade-java-guidelines[follow this guidelines].
+
+NOTE: At this moment, it is not required to upgrade the JVM version in order to run a Jenkins core more recent than ``2.164+``footnote:[It was possible to run Jenkins on Java 11 since the version `2.155` but its usage was then more complex. We do not recommend to use any version prior to `2.164`.].
+
+For the sake of simplicity, we will describe next how to run Jenkins on Java 11 with the most recent version of Jenkins.
+
+NOTE: We do encourage you to only use LTS versions for production purposes. 
+At the time of this writing, there is no LTS version with Java 11 Support.
+
+== Running Jenkins with Docker
+
+The easiest way to run Jenkins on Java 11 is to use the Docker image.
+All these images are based on the official link:https://hub.docker.com/r/_/openjdk/[openjdk:11-jdk] image maintained by the Docker Community.
+
+You can choose these Java 11 based images by using the `jdk11` tag (for the latest weekly release) or by suffixing the tag to use with `-jdk11` (eg. `2.164-jdk11`).
+
+This is the command to use, in order to start a Jenkins running on Java 11: 
+
+[source, shell]
+----
+docker pull jenkins/jenkins:jdk11
+docker run --rm -ti \
+  -p 8080:8080 -p 50000:50000
+  -v jenkins-home:/var/jenkins_home \
+  jenkins/jenkins:jdk11
+----
+
+== Running Jenkins with Java
+
+It is also possible to run Jenkins using the `java` command.
+
+Starting Jenkins would be:
+
+[source, shell]
+----
+java -jar jenkins.war
+----
+
+== Discovering issues with Java 11
+
+Between June 2018 and February 2019, the community performed many exploratory tests to try to discover as many issues related to Java 11 as possible.
+The result of this is that the community solved a lot of problems before announcing the support of Java 11 by Jenkins.
+However, it is still possible that some plugins are not yet compatible with Java 11.
+
+In order to track this, we have created a new link:https://wiki.jenkins.io/display/JENKINS/Known+Java+11+Compatibility+issues[Known Java 11 Compatibility Issues] Wiki page.
+
+If you discover any Java 11 incompatibilities, please link:https://wiki.jenkins.io/display/JENKINS/How+to+report+an+issue[report issues in our bugtracker].
+Please set `java11-compatibility` labels for such issues so that they automatically appear on the Wiki page and get triaged.
+
+For the security issues please use the standard link:https://jenkins.io/security/#reporting-vulnerabilities[vulnerability reporting process].
+Although we will be fixing Java 11 specific issues in public while it is in the preview, following the security process will help us to investigate impact on Java 8 users.
```

Comment:
I think that last part must now be removed, even more given now since yesterday 2.164 is the candidate for next LTS.
WDYT @oleg-nenashev @daniel-beck ?

Reply:
I removed the line entirely.